components/script/dom/webglrenderbuffer.rs

```diff
@@ -149,6 +149,15 @@ impl WebGLRenderbuffer {
 
         // FIXME: Invalidate completeness after the call
 
+        let max_size = self.upcast::<WebGLObject>()
+                           .context()
+                           .limits()
+                           .max_renderbuffer_size as i32;
+
+        if width > max_size || height > max_size {
```

Comment:
Why don't we move this check into WebGLRenderingContext::RenderbufferStorage instead? I think it makes more conceptual sense there.

Reply:
Hmm...I thought the `WebGLMessage::RenderbufferStorage` directly calls the corresponding `gl` API.
Also, `WebGLRenderingContext::RenderbufferStorage` already checks for these exact same conditions.

Reply:
If the only caller of this method already checks these conditions, then I think this change is redundant.